offline_signing/transfer.sh

```diff
@@ -93,7 +93,7 @@ upload() {
     fi
 }
 
-roll() {
+ready() {
```

Comment:
This should probably also set arm to 0% and the new roll can set it to 100%, right?

Reply:
yeah, we can do it that way if you want. I just left the old `roll` command as is, and it just sets the update percent to 100%.

Reply:
I think we do want to do that. It should be doing roughly the same thing for both amd64 and arm64, It's just that "slowly roll out" and "roll out all at once" are equivilent for arm. We shouldn't have a command prepare to rollout on amd and do the rollout on arm